--- signature.c.orig	2025-04-03 21:27:56.519514909 +0200
+++ signature.c	2025-04-03 21:33:35.030236713 +0200
@@ -5,7 +5,9 @@
 #include <openssl/evp.h>
 #include <openssl/pem.h>
 #include <openssl/crypto.h>
+#ifdef USE_OPENSSL_ENGINE
 #include <openssl/engine.h>
+#endif
 #include <openssl/x509.h>
 #include <string.h>
 
@@ -117,6 +119,7 @@
 	return TRUE;
 }
 
+#ifdef USE_OPENSSL_ENGINE
 static ENGINE *get_pkcs11_engine(GError **error)
 {
 	static ENGINE *e = NULL;
@@ -181,6 +184,7 @@
 out:
 	return e;
 }
+#endif
 
 static EVP_PKEY *load_key_file(const gchar *keyfile, GError **error)
 {
@@ -221,6 +225,7 @@
 static EVP_PKEY *load_key_pkcs11(const gchar *url, GError **error)
 {
 	EVP_PKEY *res = NULL;
+#ifdef USE_OPENSSL_ENGINE
 	GError *ierror = NULL;
 	ENGINE *e;
 
@@ -243,6 +248,7 @@
 		goto out;
 	}
 out:
+#endif
 	return res;
 }
 
@@ -345,6 +351,7 @@
 static X509 *load_cert_pkcs11(const gchar *url, GError **error)
 {
 	X509 *res = NULL;
+#ifdef USE_OPENSSL_ENGINE
 	GError *ierror = NULL;
 	ENGINE *e;
 
@@ -375,6 +382,7 @@
 	}
 	res = parms.cert;
 
+#endif
 out:
 	return res;
 }
